src/main/scala/rps/Game.scala

```diff
@@ -0,0 +1,43 @@
+package rps
+import java.util.Scanner
+import java.util.InputMismatchException
+
+object Game {
+  def moveRepr(move: Int) : String = {
```

Comment:
converting the move is a good idea 👍 
(we can do better with the return type, but we'll see it in another step)

Also, usually the convention is to put no spaces before the `:` in the result type, i.e.

```diff
-def moveRepr(move: Int) : String
+def moveRepr(move: Int): String
```